src/main/java/uk/gov/ons/ctp/integration/rhsvc/representation/CaseDTO.java

```diff
@@ -13,5 +14,6 @@
   private boolean invalid;
   private String refusalReceived;
   private String caseRef;
-  private AddressDTO address;
+  private Map<String, String> sample;
+  private Map<String, String> sampleSensitive;
```

Comment:
We've introduced the phone number here (via sampleSensitive).  This is something new. Did we want to do this ?

Reply:
No harm done. UI probably won't need to look at it, and can ignore.